Sources/StreamChatUI/Utils/ChatChannelNamer.swift

```diff
@@ -50,27 +54,26 @@ open class ChatChannelNamer {
                             : ""
```

Comment:
I think it would make sense to have the `and X more` string as a single string rather than gluing it from 2 different localized strings.

Reply:
Great suggestion, changed.